CodeEdit/Features/AppPreferences/Model/Text Editing/TextEditingPreferences.swift

```diff
@@ -8,6 +8,31 @@
 import AppKit
 import Foundation
 
+extension String.Encoding: Codable {
+    static var allAscii: [String.Encoding] = [String.Encoding.ascii, String.Encoding.nonLossyASCII]
```

Comment:
Please mind the [Code Style](https://github.com/CodeEditApp/CodeEdit/wiki/Code-Style#parameter-alignment)

```suggestion
    static var allAscii: [String.Encoding] = [
        .ascii, 
        .nonLossyASCII
    ]
```